src/gridcoin/upgrade.cpp

```diff
@@ -38,7 +38,14 @@ Upgrade::Upgrade()
     ExtractStatus.SnapshotExtractProgress = 0;
 }
 
-bool Upgrade::CheckForLatestUpdate(bool ui_dialog, std::string client_message_out)
+void Upgrade::ScheduledUpdateCheck()
+{
+    std::string VersionResponse = "";
+
+    CheckForLatestUpdate(VersionResponse);
+}
+
+bool Upgrade::CheckForLatestUpdate(std::string& client_message_out, bool ui_dialog, bool snapshotrequest)
 {
     // If testnet skip this || If the user changes this to disable while wallet running just drop out of here now. (need a way to remove items from scheduler)
     if (fTestNet || GetBoolArg("-disableupdatecheck", false))
```

Comment:
self note: add check to see if its not snapshotrequest to avoid the -disableupdatecheck from returning false!

Reply:
fixed. if snapshotrequest has occurred it'll continue on regardless if disableupdatecheck is set true.